<launch>

     <!-- send the coman robot XML to param server -->
     <param name="robot_description" command="$(find xacro)/xacro.py '$(find coman_description)/iit-coman/model/model.urdf'" />
     
     <!-- send the coman robot XML to param server -->
    <param name="robot_description_semantic" textfile="$(find coman_description)/iit-coman/model/coman.srdf" />   

    <!-- start robot state publisher -->
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher" output="screen" >

        <!--remap from="joint_states" to="coman/joints_state"/-->
        <param name="publish_frequency" type="double" value="50.0" />
    </node>
	
    <node pkg="gazebo_ros" type="spawn_model" name="spawn_urdf">

</launch>
